packages/react-components/react-divider/MIGRATION.md

```diff
@@ -1,60 +1,36 @@
 # Divider Migration
 
-## STATUS: WIP 🚧
-
-This Migration guide is a work in progress and is not yet ready for use.
-
 ## Migration from v8
 
-The existing `Separator` control supports the `content` property as well as children nodes. The `content` property has been removed and now only supports children.
+The existing v8's `Separator` control supports a very similar set of props to the one being proposed for v9's `Divider` with a few differences that are outlined below:
 
-With the new converged capabilities, both the `styles` and `theme` properties are no longer supported.
-
-Properties that are still available are the `alignContent` and `vertical` with the same acceptable values that are currently defined.
+- `alignContent` => `alignContent`
+- `content` => NOT SUPPORTED - use `children` instead
+- `styles` => NOT SUPPORTED - use new styling system via `tokens` instead
```

Comment:
Tweaks to wording: `content` is replaced by `children` (but still supported). And `styles` is still a prop, but not recommended.
```suggestion
- `content` => children of the `Divider`
- `styles` => recommended to use new theming system via `FluentProvider`
```

Reply:
`style` is a prop, not `styles`, right?